neo/app/controllers/Application.scala

```diff
@@ -41,7 +41,8 @@ class Application @Inject()(val wsClient: WSClient,
 
   val manager = system.actorOf(DataStoreManager.props(Migration_20160814.berryMeta, asterixConn, AQLGenerator, config))
 
-  Logger.logger.info("I'm initializing")
+  // TODO Logger.info cannot logout to console
+  Logger.info("I'm initializing")
```

Comment:
Fixed. Due to configuration at `logback.xml`. [Here](https://logback.qos.ch/manual/filters.html) is the reference. 
Now we can `log.info`, `log.warn`, `log.error` inside an actor that `with ActorLogging`, and `Logger.info`, `Logger.warn`, `Logger.error` in other classes. See the new commit.